src/cpp/rtps/builtin/discovery/participant/PDPServer2.hpp

```diff
@@ -158,10 +158,19 @@ class PDPServer2 : public aux::PDP
             fastrtps::rtps::WriterHistory* writer_history,
             const fastrtps::rtps::GuidPrefix_t& entity_guid_prefix);
 
-
     bool announcement_from_same_participant_in_disposals(
             const std::vector<fastrtps::rtps::CacheChange_t*>& disposals,
             const fastrtps::rtps::GuidPrefix_t& participant);
+    bool process_to_send_lists();
+
+    bool process_to_send_list_(
+            const std::vector<eprosima::fastrtps::rtps::CacheChange_t*>& send_list,
+            fastrtps::rtps::RTPSWriter* writer,
+            fastrtps::rtps::WriterHistory* history);
+
+    bool remove_change_from_history_nts_(
+            fastrtps::rtps::WriterHistory* history,
+            fastrtps::rtps::CacheChange_t* change);
```

Comment:
You may want to move these methods to private

Reply:
I'll remove the `_` and leave them in protected